documentation/src/main/markdown/properties/configuration/overview.md

```diff
@@ -1,3 +1,3 @@
-# Configuration Properties
+# Workflow Configuration Properties
 
```

Comment:
I don't have strong feelings about this, but are things like proxy settings, phone home passthrough arguments, signature scanner arguments (e.g. memory, etc.), etc. "workflow"? Elsewhere (/gettingstarted/workflow.html) we use the term workflow in a way that feels different to me.

That said, this is a pretty odd collection of properties. It seems to have been used as the "everything else" category, so I guess any name for it will be imperfect.

Reply:
Agreed. It was a bit of a stretch. Ultimately I think some additional massaging of the content under this title will be the way to go. In the immediate term this section primarily appears to provide additional information on properties that called out in other properties list so I think perhaps "Configuration Property Details" is more apt.